module/VuFindSearch/src/VuFindSearch/Backend/Solr/SearchHandler.php

```diff
@@ -409,6 +421,43 @@ protected function mungeValues($search, $tokenize = true)
         return $mungeValues;
     }
 
+    /**
+     * Custom munge search string of a dismax query.
+     *
+     * @param string $search Search string
+     *
+     * @return string
+     */
+    protected function dismaxMunge($search)
+    {
+        foreach ($this->specs['DismaxMunge'] as $operation) {
+            switch ($operation[0]) {
```

Comment:
I would strongly recommend moving this switch statement to a support method so it can be shared by the two places that do munging -- that way we can add additional munge rules without having to copy and paste them to two different spots.